extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -178,6 +264,14 @@ export class AmpLightboxGallery extends AMP.BaseElement {
 
     /** @private {?Element} */
     this.sourceElement_ = null;
+
+    /**
+     * A listener is set up to prevent carousel scrolling when we do a swipe to
+     * dismiss gesture. This is used to clean up the listener when no longer
+     * needed.
+     * @private {?function()}
+     */
+    this.preventCarouselScrollUnlistener_ = null;
```

Comment:
Do we also prevent accidental dismissal mid carousel swipe?

Reply:
Since we use a y swipe recognizer, as soon as you swipe horizontally, swipe to dismiss won't be started until you release your current swipe.